config/locales/client.en.yml

```diff
@@ -1924,21 +1924,36 @@ en:
 
       actions:
         recover: "Un-Delete Topic"
+        recover_tooltip: "Restores the topic."
         delete: "Delete Topic"
+        delete_tooltip: "Marks the topic as deleted."
         open: "Open Topic"
+        open_tooltip: "Marks the topic as open and allows new replies."
         close: "Close Topic"
+        close_tooltip: "Marks the topic as closed and prevents new replies."
         multi_select: "Select Posts…"
+        multi_select_tooltip: "Toggles multiselect mode in which multiple posts can be selected."
```

Comment:
I don't think we ever mention this as the "multiselect mode" anywhere else.

Reply:
I just took the same name the locale variable has. But I think, "multiselect mode" is pretty easy to understand :) Other ideas on how to call it?